output.go

```diff
@@ -44,12 +44,22 @@ const (
 	DefaultLogName = "jasper"
 )
 
+type LogFormat string
+
+const (
+	LogFormatPlain   = ""
```

Comment:
we should make this something more than the empty string

Reply:
Changed to "plain".